.travis.yml

```diff
@@ -3,9 +3,6 @@ language: php
 sudo: false
 
 php:
-  - 5.4
```

Comment:
Why? We currently have support for 5.x in 1.6, don't want to remove it if not required.

Reply:
Besides this, LGTM.

Reply:
You are correct. This will get fixed.

Reply:
Note, also in composer.json